app/storage/questionnaire_storage.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from app.data_model.database import QuestionnaireState, commit_or_rollback
+from app.data_model.database import db_session
+
+logger = logging.getLogger(__name__)
+
+
+class QuestionnaireStorage:
```

Comment:
It stores quite a lot to be fair, it stores the users metadata, completed block and answers. If you can think of an apt name for that I am happy to change